### PR TITLE
fuzz: add target for DecodeBase64

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/6050

Previous PR: https://github.com/OISF/suricata/pull/11002

Changes since v10:
- rebased

Note: Running 10+ mins on my system w/o any issues.

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 20537

**Comment by victorjulien** [comment]
@catenacyber I'll approve it if you think it's good.

File: src/Makefile.am [hunk 1]
```diff
@@ -1414,6 +1414,17 @@ endif
 nodist_EXTRA_fuzz_predefpcap_aware_SOURCES = force-cxx-linking.cxx
 endif
 
+fuzz_decodebase64_SOURCES = tests/fuzz/fuzz_decodebase64.c
+fuzz_decodebase64_LDFLAGS = $(LDFLAGS_FUZZ)
+fuzz_decodebase64_LDADD = $(LDADD_FUZZ)
+if HAS_FUZZLDFLAGS
```
Comment (catenacyber) [inline src/Makefile.am:1420]: You could pick the fix of https://github.com/OISF/suricata/pull/11036/commits/964a38493f07696fd889f4907ee0dc2ff20e1d4b for this target already ;-)

**Review by catenacyber — APPROVED** [review]
Looks good to me, did not see bugs in the first few minutes of running

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #11072, thanks!

**Pull request closed without merging**